samples/subsys/power/stm32_pm/lptim/sample.yaml

```diff
@@ -0,0 +1,12 @@
+sample:
```

Comment:
Please add a README file

Reply:
through a "DEMO_DESCRIPTION" in the main.c